packages/hardhat-deployer/README.md

```diff
@@ -0,0 +1,125 @@
+# hardhat-superfluid
+
+This plugin brings a Superfluid framework deployer for development environments using `ethers`.
+
+## What does it do?
+
+The `hardhat-superfluid` plugin handles deployment and initialization of Superfluid contracts in a
+development environment.
+
+## Installation
+
+```bash
+npm install hardhat-superfluid @superfluid-finance/ethereum-contracts@1.2.2
+```
+
+Import the plugin in `hardhat.config.js`:
+
+```js
+require("hardhat-superfluid");
+```
+
+Or in a Typescript file, use the following in `hardhat.config.ts`:
+
+```ts
+import "hardhat-superfluid";
+```
+
+## Required plugins
+
+-   [@nomiclabs/hardhat-ethers](https://github.com/nomiclabs/hardhat/tree/master/packages/hardhat-ethers)
+
+## Tasks
+
+This plugin creates no additional tasks.
+
+## Environment extensions
+
+This plugin extends the Hardhat Runtime Environment by adding the `superfluidFrameworkDeployer`
+object, of type `SuperfluidFrameworkDeployer`. See "Usage" for details.
+
+## Usage
+
+> **NOTICE**: The deployments' gas limits are set to 6 million each. The default Hardhat network
```

Comment:
This is not meant to be used on a public network. This project uses `SuperfluidFrameworkDeployer` instead of running each transaction manually for maintainability reasons.

The `SuperfluidFrameworkDeployer` is not suitable for production.